### PR TITLE
增加jvm命令中memory可读性

### PR DESCRIPTION
**Pull request by bohrqiu**
原输出：
image

新输出：
image

### CODE REVIEW

File: core/src/main/java/com/taobao/arthas/core/util/StringUtils.java [hunk 1]
```diff
@@ -872,4 +874,16 @@ public static String classLoaderHash(Class<?> clazz) {
         if (clazz == null || clazz.getClassLoader() == null) return "null";
         return Integer.toHexString(clazz.getClassLoader().hashCode());
     }
+
+    /**
+     * format byte size to human readable format
+     * @param bytes byets
+     * @return  human readable format
+     */
+    public static String humanReadableByteCount(long bytes) {
```
Comment (ralf0131) [inline core/src/main/java/com/taobao/arthas/core/util/StringUtils.java]: Hi, I would suggest to add some unit test to this method.

**Comment by ralf0131** [comment]
Hi, 

in the pull request description, please use text instead of pics. Pics are not searchable.

**Comment by bohrqiu** [comment]
1. add `jvm` command human readable memory size.
2. add `DEADLOCK-COUNT`  for thread metrics
3. fix wrong metrics name `LIVE-COUNT`  to `PEAK-COUNT`

File: core/src/main/java/com/taobao/arthas/core/util/StringUtils.java (hunk 1 above)
Comment (bohrqiu) [reply core/src/main/java/com/taobao/arthas/core/util/StringUtils.java]: 本来准备写测试用例，但是发现这个项目基本上没有测试用例😓

Comment (ralf0131) [reply core/src/main/java/com/taobao/arthas/core/util/StringUtils.java]: Yes. I agree that the test cases should be improved...

**Pull request merged**